src/deluge/storage/flash_storage.cpp

```diff
@@ -197,6 +202,9 @@ bool gridEmptyPadsCreateRec;
 bool gridAllowGreenSelection;
 GridDefaultActiveMode defaultGridActiveMode;
 
+
+enum defaultSliceMode;
```

Comment:
This is declaring an enum named defaultSliceMode

What you want is a SampleRepeatMode named defaultSliceMode

Reply:
and now i know what an enum is.

Reply:
lets call it an uphill battle